packages/preview2-shim/lib/browser-async/filesystem.js

```
@@ -0,0 +1,291 @@
+    upper += BigInt(this.#mtime);
+    return { upper, lower: BigInt(0) };
+  }
+
+  metadataHashAt(_pathFlags, _path) {
+    let upper = BigInt(0);
+    upper += BigInt(this.#mtime);
+    return { upper, lower: BigInt(0) };
+  }
+}
+const descriptorGetEntry = Descriptor.prototype._getEntry;
+delete Descriptor.prototype._getEntry;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)